website/_layouts/home_layout.html

```diff
@@ -22,14 +22,20 @@
         }
 
         $.ajax({
-            url: 'https://api.ab2d.cms.gov/health',
+            url: 'https://pk3dch6jka.execute-api.us-east-1.amazonaws.com/status/ab2dapi',
```

Comment:
Would this be better as an Env Var so we could have one for SBX and one for PROD?

Reply:
Yes this is a good idea. I think we can change it in future work when. I'll make a note of it